openstudiocore/src/resultsviewer/MainWindow.cpp

```diff
@@ -1394,6 +1395,7 @@ namespace resultsviewer{
     {
       for (QString file : fileList)
       {
+        file = QString::fromStdString(openstudio::filesystem::canonical(toPath(file)).string());
```

Comment:
@kbenne need to be careful with canonical since i think it will throw if the file does not exist:

http://en.cppreference.com/w/cpp/experimental/fs/canonical

I ended up writing this function in WorkflowJSON:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/filetypes/WorkflowJSON.cpp#L53